Library/Formula/consul.rb

```diff
@@ -13,6 +13,8 @@ class Consul < Formula
     sha256 "67052157266d536598ba7eb5af31a10fd3e609ca812be6d50869d0fe80a583f9" => :mountain_lion
   end
 
+  option "with-web-ui"
```

Comment:
Could you add a second parameter with a description of what the option does?  It can me something simple; like:
```ruby
option "with-web-ui", "Build GUI"
```